wally-registry-frontend/src/pages/Package.tsx

```diff
@@ -173,150 +179,188 @@ export default function Package() {
-                </MetaItem>
-
-                {packageMetadata?.package.license && (
-                  <MetaItem title="License" width="half">
-                    <a
-                      href={`https://choosealicense.com/licenses/${packageMetadata?.package.license.toLocaleLowerCase()}`}
+        <FlexColumns>
+          <WideColumn>
+            <Heading>{packageName}</Heading>
+
+            <Paragraph>
+              {packageMetadata?.package.description
+                ? packageMetadata.package.description
+                : `${capitalize(packageName)} has no provided description.`}
+            </Paragraph>
+          </WideColumn>
+
+          <NarrowColumn>
+            <MetaHeader>Metadata</MetaHeader>
+
+            {packageMetadata?.package && (
+              <MetaItem title="Install" width="full">
+                <CopyCode
+                  packageName={packageMetadata.package.name}
+                  version={packageMetadata.package.version}
+                />
+              </MetaItem>
+            )}
+
+            <MetaItem title="Version" width="half">
+              <select
+                name="version"
+                id="version-select"
+                value={packageVersion ?? "?.?.?"}
+                onChange={(a) => {
+                  hist.push(
+                    `/package/${packageScope}/${packageName}?version=${a.target.value}`
+                  )
+                }}
+              >
+                {packageHistory?.map((item: WallyPackageMetadata) => {
+                  return (
+                    <option
+                      key={item.package.version}
+                      value={item.package.version}
                     >
-                      {packageMetadata?.package.license}
-                    </a>
-                  </MetaItem>
-                )}
-
-                {packageMetadata?.package.realm && (
-                  <MetaItem title="Realm" width="half">
-                    {capitalize(packageMetadata?.package.realm)}
-                  </MetaItem>
-                )}
-
-                {/* TODO: Re-implement when Wally API supports custom source repos */}
-                {/* {packageMetadata?.package.registry && (
+                      {item.package.version}
+                    </option>
+                  )
+                })}
+              </select>
+            </MetaItem>
+
+            {packageMetadata?.package.license && (
+              <MetaItem title="License" width="half">
+                <a
+                  href={`https://choosealicense.com/licenses/${packageMetadata?.package.license.toLocaleLowerCase()}`}
+                >
+                  {packageMetadata?.package.license}
+                </a>
+              </MetaItem>
+            )}
+
+            {packageMetadata?.package.realm && (
+              <MetaItem title="Realm" width="half">
+                {capitalize(packageMetadata?.package.realm)}
+              </MetaItem>
+            )}
```

Comment:
I don't think the realm can be null here, so it should be safe to eliminate the guard and simplify.
```suggestion
            <MetaItem title="Realm" width="half">
              {capitalize(packageMetadata.package.realm)}
            </MetaItem>
```